.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "proto"]
 	path = proto
-	url = https://github.com/axelarnetwork/grpc-protobuf.git
+	url = https://github.com/mobilecoinofficial/axelar-grpc-protobuf.git
```

Comment:
This will be reverted once https://github.com/axelarnetwork/grpc-protobuf/pull/13 is merged